cli/cmd/doc.go

```diff
@@ -7,40 +7,58 @@ import (
 	"strings"
 
 	"github.com/spf13/cobra"
-	"github.com/spf13/cobra/doc"
+	cobradoc "github.com/spf13/cobra/doc"
 	"sigs.k8s.io/yaml"
+
+	"github.com/linkerd/linkerd2/pkg/k8s"
 )
 
+type references struct {
+	CLIReference         []cmdDoc
+	AnnotationsReference []annotationDoc
+}
+
 type cmdOption struct {
 	Name         string
-	Shorthand    string `yaml:",omitempty"`
-	DefaultValue string `yaml:"default_value,omitempty"`
-	Usage        string `yaml:",omitempty"`
+	Shorthand    string
```

Comment:
Looks like it is kinda legacy from other `yaml` package and these tags are not used by `"sigs.k8s.io/yaml"`

Reply:
ah, it's unintentional that these options are not being applied.  it looks like we currently have a bunch of `SeeAlso: null` in the output, for example.  we should look into how to omit these fields in "sigs.k8s.io/yaml".

Reply:
I've investigated a bit and `"sigs.k8s.io/yaml"` accept json-tags.
So here should be `json:",omitempty"` and so on to make it work. As for me it'll look confusingly.

Reply:
> I've investigated a bit and "sigs.k8s.io/yaml" accept json-tags.

Yeah, my jaw dropped when I first found out about it, because it first converts everything to JSON and then into YAML.